unity-client/Assets/Scripts/Tests/CursorControllerTests/CursorControllerTests.cs

```diff
@@ -88,6 +90,79 @@ public IEnumerator OnPointerHoverFeedbackIsDisplayedCorrectly()
 
             yield return null;
 
+            // Check cursor shows normal sprite
+            Assert.AreEqual(cursorController.cursorImage.sprite, cursorController.normalCursor);
+
+            DCLCharacterController.i.ResumeGravity();
+        }
+
+        [UnityTest]
+        public IEnumerator OnPointerHoverFeedbackNotDisplayedOnInvisibles()
+        {
+            DecentralandEntity entity;
+            BoxShape shape;
+
+            shape = TestHelpers.InstantiateEntityWithShape<BoxShape, BoxShape.Model>(
+                scene,
+                DCL.Models.CLASS_ID.BOX_SHAPE,
+                Vector3.zero,
+                out entity,
+                new BoxShape.Model() { });
+
+            TestHelpers.SetEntityTransform(scene, entity, new Vector3(8, 2, 10), Quaternion.identity, new Vector3(3, 3, 3));
+            yield return shape.routine;
+
+            var OnPointerDownModel = new OnPointerDown.Model()
+            {
+                type = OnPointerDown.NAME,
+                uuid = "pointerevent-1"
+            };
+
+            var component = TestHelpers.EntityComponentCreate<OnPointerDown, OnPointerDown.Model>(scene, entity,
+                OnPointerDownModel, CLASS_ID_COMPONENT.UUID_CALLBACK);
+
+            Assert.IsTrue(component != null);
+
+            yield return null;
+
+            var cursorController = GameObject.FindObjectOfType<CursorController>();
+
+            // Check cursor shows normal sprite
+            Assert.AreEqual(cursorController.cursorImage.sprite, cursorController.normalCursor);
+
+            DCLCharacterController.i.PauseGravity();
+            DCLCharacterController.i.SetPosition(new Vector3(8, 1, 7f));
+
+            var cameraController = GameObject.FindObjectOfType<CameraController>();
```

Comment:
`cameraController` is already part of `TestsBase`. Maybe this line isn't necessary.